care/emr/resources/scheduling/schedule/spec.py

```diff
@@ -77,6 +78,12 @@ def validate_for_slot_type(self):
                 end_time = datetime.datetime.combine(
                     datetime.datetime.now(tz=UTC).date(), availability.end_time
                 )
+                total_time = (end_time - start_time).total_seconds() / 60
+                total_slots = (
+                    total_time / self.slot_size_in_minutes
+                ) * self.tokens_per_slot
```

Comment:
why multiply by tokens per slot? 🤔 

number of slots per availability has nothing to do with tokens per slot right?

we don't want to restrict number of tokens, we want to restrict number of slots.